Firestore/core/src/firebase/firestore/model/field_value.cc

```diff
@@ -343,6 +377,9 @@ void FieldValue::SwitchTo(const Type type) {
     case Type::Blob:
       blob_value_.~vector();
       break;
+    case Type::Reference:
+      reference_value_.reference.~DocumentKey();
```

Comment:
Why not do `reference_value_.~DatabaseReference();`?

Reply:
done